src/bepasty/app.py

```diff
@@ -45,13 +46,29 @@ def __call__(self, environ, start_response):
             return ['This URL does not belong to the bepasty app.'.encode()]
 
 
+def setup_secret_key(app):
+    """
+    to detect the configuration change, and make obsolete session for
+    old config, this add unique id to SECRET_KEY additionally.
+
+    NOTE: for now, only permissions is checked
```

Comment:
Maybe rephrase this:

The secret key is used to sign cookies and cookies not signed with the current secret key are considered invalid.

Here, we amend the configured secret key, so it depends on some other config values. Changing any of these values will change the computed secret key (and thus invalidate all previously made cookies).

Currently supported secret-changing config values: PERMISSIONS